integrationTests/p2p/networkSharding-hbv2/networkSharding_test.go

```diff
@@ -212,6 +212,7 @@ func testUnknownSeederPeers(
 
 	for _, nodes := range nodesMap {
 		for _, n := range nodes {
+			assert.Equal(t, 0, len(n.Messenger.GetConnectedPeersInfo().UnknownPeers))
```

Comment:
:+1: :+1: :+1: 
this proves the fix will eventually make the unknown list empty